community/samples/serving/helloworld-r/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use the official Python image.
+# https://hub.docker.com/_/python
+FROM python:3.7
+
+# Copy local code to the container image.
+ENV APP_HOME /app
+WORKDIR $APP_HOME
+COPY . .
+
+# Install production dependencies.
+RUN pip install Flask gunicorn
+
+# Install R
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends --allow-unauthenticated r-base
```

Comment:
Why do you need `--allow-unauthenticated`?

Reply:
not necessary. removed.